### ISSUE TITLE
Advanced Search Tab Re-design 

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#418]
**Problem Description:**
LUX would like to redesign the front end layout based on POW's mockups. This issue will support the re-design of the front end layout (#270) and specifically focus on the advanced search tab re-design. 

Prerequisite: https://github.com/project-lux/lux-frontend/issues/416

**Expected Behavior/Solution:**
LUX advanced search tabs to be re-configured to match the requirements and mockups below. 

**Requirements:**

- Merge the advanced search query with the HELP text. 
- - Query will remain on the left hand side
- - Help text will remain on the right hand side
- - a grey divider will be placed between the two sections. 
- Tabs will be moved to be above the new merged section.
- Tabs will be connected to the advanced search query. 
- Tabs will contain their respective icon to the right side.
- `X results` will be placed underneath the <Scope Label> 
- Icons will always appear for all scope tabs.
- The new merged facet & search results block will live underneath the new merged query block. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- https://lux.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&qt=objects

**Dependencies/Blocks:**
Prerequisite: https://github.com/project-lux/lux-frontend/issues/416

**Related Github Issues:**
- META: https://github.com/project-lux/lux-frontend/issues/270

**Related links:**
[Presentation](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=3377%3A9485&node-id=3377-9572&viewport=4999%2C493%2C0.26&t=Axk0QSGESY3qeZh1-1&scaling=scale-down&content-scaling=fixed)
[Design walkthrough](https://www.figma.com/design/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?node-id=30-6364&node-type=canvas&t=Sy4U7OxDEOrthtNY-0)
[Prototype](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=182%3A6364&node-id=3317-20738&viewport=-5411%2C-5010%2C0.36&t=yi4XGHaZfpsKRsqT-9&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=3317%3A20738&show-proto-sidebar=1)

**Wireframe/Mockup:**
![image](https://github.com/user-attachments/assets/aaf4994a-5bdb-4533-bfa0-b984786762f2)

**Comment by prowns** [comment]
same comment as on #417 

@kamerynB - can you validate the grays used in the tabs? It seems to be a much darker gray than I'm seeing deployed.

Also - are you planning to add the Object/Works (?) overlay as part of this ticket, or is that separate?

**Comment by roamye** [comment]
1. (minor issue) | the people and groups icon, as shown above is inconsistent with where the other icons are placed. This also affects the "66 results" as it is placed in the middle. 

Also noted the above in https://github.com/project-lux/lux-frontend/issues/417

**Comment by roamye** [comment]
Approved by UAT but needs a separate ticket to address the issues below: 

- The shadow of the tabs is different than the shadow for the advanced search query. 
- Help box does not fit properly. 
- the line from the help text does not continue on. 
- the additional of the (?)/overlay. 

Example: https://lux-front-tst.collections.yale.edu/view/results/events?q=%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D&qt=concepts (Zoom in to 200% to see the issues)

**Comment by roamye** [comment]
Closing, looks good in PROD. 

Created - https://github.com/project-lux/lux-other/issues/12 to tackle the redesign of the help text box. Other issues were addressed in the hot fix.
